src/allencell_ml_segmenter/main/main_model.py

```diff
@@ -82,3 +82,6 @@ def set_selected_channels(
         ):
             self._selected_channels = new_channels
             self.signals.selected_channels_changed.emit()
+
+    def training_complete(self) -> None:
+        self.dispatch(Event.PROCESS_TRAINING_COMPLETE)
```

Comment:
Is there not already an event being dispatched when training is complete?  I would think that there is because of the progress dialog behavior?